test/cross-chain/cross-chain.js

```diff
@@ -0,0 +1,156 @@
+/* globals artifacts */
+const fs = require("fs");
+const chai = require("chai");
+const bnChai = require("bn-chai");
+const { ethers } = require("ethers");
+
+const Promise = require("bluebird");
+
+const exec = Promise.promisify(require("child_process").exec);
+
+const { expect } = chai;
+chai.use(bnChai(web3.utils.BN));
+
+const IColonyNetwork = artifacts.require("IColonyNetwork");
+const Token = artifacts.require("Token");
+const IColony = artifacts.require("IColony");
+
+const setupBridging = require("../../scripts/start-bridging-environment");
+
+contract("Cross-chain", () => {
+  let colony;
+  let homeBridge;
+  let foreignBridge;
+  let gnosisSafe;
+  let zodiacBridge;
+  let bridgeMonitor;
+
+  const ADDRESS_ZERO = ethers.constants.AddressZero;
+
+  const TRUFFLE_PORT = process.env.SOLIDITY_COVERAGE ? 8555 : 8545;
+  const OTHER_RPC_PORT = 8546;
+
+  const HOME_PORT = process.env.TRUFFLE_FOREIGN === "true" ? OTHER_RPC_PORT : TRUFFLE_PORT;
+  const FOREIGN_PORT = process.env.TRUFFLE_FOREIGN === "true" ? TRUFFLE_PORT : OTHER_RPC_PORT;
+
+  const foreignRpcUrl = `http://127.0.0.1:${FOREIGN_PORT}`;
+  const homeRpcUrl = `http://127.0.0.1:${HOME_PORT}`;
+
+  const ethersForeignSigner = new ethers.providers.JsonRpcProvider(foreignRpcUrl).getSigner();
+  const ethersHomeSigner = new ethers.providers.JsonRpcProvider(homeRpcUrl).getSigner();
+
+  before(async () => {
+    await exec(
```

Comment:
Couldn't this be an invocation of `start-bridging-environment.sh` with different arguments?